sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationOptionsExt.kt

```diff
@@ -0,0 +1,46 @@
+@file:JvmName("ViewAnnotationOptionsExt")
+package com.mapbox.maps.viewannotation
+
+import android.graphics.Rect
+import androidx.annotation.VisibleForTesting
+import com.mapbox.maps.ViewAnnotationAnchor
+import com.mapbox.maps.ViewAnnotationOptions
+
+/**
+ * Get [Rect] from [ViewAnnotationOptions]'s geometry, width and height.
+ * This function takes [ViewAnnotationOptions.geometry] as the center of rectangle and
+ * use width, height and offset values to calculate [Rect] associated.
+ *
+ * @return [Rect] associated with [ViewAnnotationOptions]
+ */
+@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+internal fun ViewAnnotationOptions.frame(): Rect? {
+  if (width != null && height != null) {
+    val offsetWidth = if (width!! > 0) (width!! * 0.5).toInt() else 0
+    val offsetHeight = if (height!! > 0) (height!! * 0.5).toInt() else 0
```

Comment:
So for views that are not yet measured (width / height = -1) we'll have Rect(0,0,0,0). Is that intentional? Maybe return `null` for such cases?

Reply:
These are for the calculations of paddings. if width/height is -1 we will use 0 paddings for those. I have added docs for this.

Reply:
Right, so it'll return Rect with width and height == 0, right?